tests/python/integration/test_tuning.py

```diff
@@ -274,10 +312,13 @@ def __init__(
             do_fork=False,
             runtime=None,
         ):
+            # pylint: disable=too-many-function-args
             super().__init__(timeout, n_parallel, build_kwargs, build_func, do_fork, runtime)
+
+            # pylint: disable=too-many-function-args
```

Comment:
Out of curiosity, here number of args are not > 5. Why does it complain? Any clue?

Reply:
that one was superfluous, removed